src/otx/algorithms/segmentation/adapters/mmseg/configurer.py

```diff
@@ -202,89 +197,27 @@ def configure_input_size(
         logger.info("Input size is changed to {}".format(input_size))
 
 
-class IncrSegmentationConfigurer(SegmentationConfigurer):
+class IncrSegmentationConfigurer(IncrConfigurerMixin, SegmentationConfigurer):
     """Patch config to support incremental learning for semantic segmentation."""
 
-    def configure_task(self, cfg: ConfigDict) -> None:
-        """Patch config to support incremental learning."""
-        super().configure_task(cfg)
-
+    def is_incremental(self) -> bool:
+        """Return whether current model classes is increased from original model classes."""
         # TODO: Revisit this part when removing bg label -> it should be 1 because of 'background' label
         if len(set(self.org_model_classes) & set(self.model_classes)) == 1 or set(self.org_model_classes) == set(
             self.model_classes
         ):
             is_cls_incr = False
         else:
             is_cls_incr = True
-
-        # Update TaskAdaptHook (use incremental sampler)
-        task_adapt_hook = ConfigDict(
-            type="TaskAdaptHook",
-            src_classes=self.org_model_classes,
-            dst_classes=self.model_classes,
-            model_type=cfg.model.type,
-            sampler_flag=is_cls_incr,
-            efficient_mode=cfg["task_adapt"].get("efficient_mode", False),
-        )
-        update_or_add_custom_hook(cfg, task_adapt_hook)
+        return is_cls_incr
```

Comment:
```suggestion
        # TODO: Revisit this part when removing bg label -> it should be 1 because of 'background' label
        return (
            len(set(self.org_model_classes) & set(self.model_classes)) > 1
            and set(self.org_model_classes) != set(self.model_classes)
        )
```
How about to change code like this?